utils/src/main/scala/za/co/absa/enceladus/utils/validation/field/FieldValidator.scala

```diff
@@ -33,15 +33,15 @@ object FieldValidator {
   def validate(field: StructField): Seq[ValidationIssue] = {
     val validator = field.dataType match {
       case _: DateType =>
-        Some(new FieldValidatorDate)
+        Option(FieldValidatorDate)
```

Comment:
I know I'm the one who brought this practice up (wrapping things in `Option` rather than `Some`), but I must admit it looks weird when applied to concrete instances. I think it might actually throw doubt on wether or not a constructor will return a value or null, which is a ridiculous overhead in terms of cognitive complexity. 

I'd suggest to take it back to `Some` but use `Option` when passing in a variable.

Reply:
I will change it here, maybe elsewhere too if I stumble upon such code. But will not scan the whole code 😉

Reply:
Hm talked to the guy who mentored me on Scala. He's also for `Option(x)` rather then `Some(x)`. Also because of better type inference. I checked on it, and indeed. So I think I stick to it.